[2024-04-05 19:36:51,737] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2024-02-26T00:00:00+00:00 [queued]>
[2024-04-05 19:36:51,742] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2024-02-26T00:00:00+00:00 [queued]>
[2024-04-05 19:36:51,742] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-04-05 19:36:51,742] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-04-05 19:36:51,742] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-04-05 19:36:51,760] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2024-02-26 00:00:00+00:00
[2024-04-05 19:36:51,763] {standard_task_runner.py:52} INFO - Started process 726213 to run task
[2024-04-05 19:36:51,765] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'extrai_dados', 'scheduled__2024-02-26T00:00:00+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/extrai_dados_climaticos.py', '--cfg-path', '/tmp/tmpmg86d9a_', '--error-file', '/tmp/tmpn1sex4um']
[2024-04-05 19:36:51,766] {standard_task_runner.py:80} INFO - Job 122: Subtask extrai_dados
[2024-04-05 19:36:51,801] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.extrai_dados scheduled__2024-02-26T00:00:00+00:00 [running]> on host TEC017860.
[2024-04-05 19:36:51,852] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2024-02-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-02-26T00:00:00+00:00
[2024-04-05 19:36:52,464] {python.py:173} INFO - Done. Returned value was: None
[2024-04-05 19:36:52,468] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=extrai_dados, execution_date=20240226T000000, start_date=20240405T223651, end_date=20240405T223652
[2024-04-05 19:36:52,520] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-05 19:36:52,530] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
